packages/govern-lib/src/config.ts

```diff
@@ -0,0 +1,27 @@
+import { ChainId, Network } from './types'
+
+export const networkContexts = new Map<
```

Comment:
I would just call this ``config`` or ``networks`` and rename the file.

Reply:
True let’s do this. I was trying to make the separation between network and network + subgraph (i.e. network context) more clear, but it is probably unnecessary for this file yes :+1: